src/page/template/panel/guest-options.htm

```diff
@@ -5,42 +5,40 @@
       <div class="panel__header__title" data-bind="text: t('guestOptionsTitle')"></div>
       <close-icon class="right-panel-close icon-button" data-bind="clickOrDrag: onClose" data-uie-name="do-close"></close-icon>
     </div>
-    <div class="panel__content">
-      <div class="panel__content__scroll" data-bind="antiscroll: shouldUpdateScrollbar">
```

Comment:
@AndyLnd could you check if this `shouldUpdateScrollbar` is still used and remove it from the view model if it is not.